### PR TITLE
Update workaround for wasm hosted apps

### PR DESCRIPTION
**Pull request by thaystg**
To document what was discussed here: https://github.com/dotnet/runtime/issues/70841

### CODE REVIEW

**Comment by radical** [comment]
@TanayParikh Is anything else needed here?

File: release-notes/7.0/known-issues.md [hunk 1]
```diff
@@ -8,7 +8,20 @@ You may encounter the following known issues, which may include workarounds, mit
 
 It’s not possible to debug a Blazor app using .net7 preview 5 https://github.com/dotnet/runtime/pull/70383
 
-Workaround:
+Workaround for a hosted app:
```
Comment (radical) [inline release-notes/7.0/known-issues.md:11]:
```suggestion
#### Workaround for a hosted app:
```

File: release-notes/7.0/known-issues.md [hunk 2]
```diff
@@ -8,7 +8,20 @@ You may encounter the following known issues, which may include workarounds, mit
...
+    <ItemGroup>
+        <PackageReference Include="Serilog.Extensions.Logging.File" Version="2.0.0" ExcludeAssets="all" GeneratePathProperty="true"/>
+    </ItemGroup>
+    <Target Name="_CopySerilogDeps" AfterTargets="Build">
+        <Copy SourceFiles="$(PkgSerilog_Extensions_Logging_File)\lib\netstandard2.0\Serilog.Extensions.Logging.File.dll"
+	        DestinationFolder="$(OutputPath)\BlazorDebugProxy"/>
+    </Target>
+```
+
+Workaround for a non-hosted app:
```
Comment (radical) [inline release-notes/7.0/known-issues.md:24]:
```suggestion
#### Workaround for a non-hosted app:
```

**Review by TanayParikh — CHANGES_REQUESTED** [review]

**Comment by TanayParikh** [comment]
Thanks @thaystg!

**Pull request merged**